package.json

```diff
@@ -35,15 +35,15 @@
     "dayjs": "1.10.6",
     "dotenv": "10.0.0",
     "eslint-config-airbnb": "18.2.1",
-    "nightwatch": "1.7.7",
-    "nightwatch-vrt": "0.2.10",
+    "nightwatch-vrt": "github:ginnyyadav/nightwatch-vrt",
```

Comment:
If this is going to be something that needs to be maintained long-term, it may make sense to transfer it to `tidepool-org`

Also, usually a good idea to still lock down a version.  With github URLs, you can append a "commit-ish" string to the end to have it do this.  i.e.
```
"nightwatch-vrt": "github:ginnyyadav/nightwatch-vrt#MY_COMMIT_ID",
"nightwatch-vrt": "github:ginnyyadav/nightwatch-vrt#MY_TAG_OR_BRANCH",
```

Reply:
Oh thats a great piece of information. thank you!